urllib3/contrib/pyopenssl.py

```diff
@@ -40,15 +43,12 @@
 """
 from __future__ import absolute_import
 
-try:
-    from ndg.httpsclient.ssl_peer_verification import SUBJ_ALT_NAME_SUPPORT
-    from ndg.httpsclient.subj_alt_name import SubjectAltName as BaseSubjectAltName
-except SyntaxError as e:
-    raise ImportError(e)
-
+import idna
 import OpenSSL.SSL
-from pyasn1.codec.der import decoder as der_decoder
-from pyasn1.type import univ, constraint
+from cryptography import x509
+from cryptography.haztmat.backends.openssl import backend as openssl_backend
```

Comment:
Yup, and a typo that meant that our tests weren't running on this code. Good spot!